keyboards/handwired/sauceball/info.json

```
@@ -0,0 +1,38 @@
+{
+    "manufacturer": "xia0",
+    "keyboard_name": "sauceball",
+    "maintainer": "xia0",
+    "bootloader": "halfkay",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "debug_mouse": true,
+        "debug_enable": true
```

With the process here being `feature.to_uppercase` + `_ENABLE`, the resulting `DEBUG_MOUSE_ENABLE` and `DEBUG_ENABLE_ENABLE` are not valid options.

```suggestion
        "nkro": true
```